packages/cashier/src/pages/withdrawal/disable-withdrawal-modal.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+
+const DisableWithdrawalModal = ({ is_risk_client, onClick }) => {
+    return (
+        <Modal is_open={is_risk_client} small className='center-risk-modal' title={localize('Financial Assessment')}>
+            <Modal.Body>
+                <Text as='p' size='s' align='center'>
```

Comment:
`s` is default text size.
```suggestion
                <Text as='p' align='center'>
```